Sources/StreamChat/Models/Attachments/ChatMessageAttachment.swift

```diff
@@ -4,6 +4,39 @@
 
 import Foundation
 
+public struct StreamAttachment<Payload> {
```

Comment:
The Stream prefix could be a bit generic, and it does not give enough context on how this is used. What about this:
```suggestion
/// An attachment that is not assigned to any message or channel.
public struct StandaloneAttachment<Payload> {
```